packages/firebase_analytics/firebase_analytics/example/lib/main.dart

```diff
@@ -72,7 +72,7 @@ class _MyHomePageState extends State<MyHomePage> {
         'int': 42,
         'long': 12345678910,
         'double': 42.0,
-        'bool': true,
+        'bool': true.toString(),
```

Comment:
Do you mind making a note that only numbers and strings are accepted (hence why stringified)? Also paste in this link perhaps: https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets#overview

Reply:
also point to master branch please.

Reply:
> Do you mind making a note that only numbers and strings are accepted (hence why stringified)? Also paste in this link perhaps: https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets#overview

What do you mean about _making a note_?

Reply:
a comment: 
```dart
    'int': 42,
        'long': 12345678910,
        'double': 42.0,
        'bool': true,
		// comment in code
        'bool': true.toString(),

```

Reply:
> also point to master branch please.

I'm new in github, do I need to open a different request or I can point to the master from this one?

Reply:
point to master on this PR.

Reply:
> point to master on this PR.

How do I do this from this PR?